packages/zipkin/test/model.test.js

```diff
@@ -56,4 +56,26 @@ describe('Span setters', () => {
 
     expect(span.duration).to.equal(1);
   });
+
+  it('should convert annotation values to strings', () => {
+    const span = new Span(new TraceId({
+      traceId: new Some('a'),
+      spanId: 'b',
+    }));
+    span.addAnnotation(101239, 10);
+
+    expect(span.annotations[0].value).to.equal('10');
+  });
+
+  it('should convert tag values to strings', () => {
```

Comment:
I would add cases for array and object as cases (maybe not checking the output but checking that it does not break). If it helps, check what we do in https://github.com/openzipkin/zipkin-js/pull/239/files#diff-69d935ef65c95d1891e05297e13f46aaR121.

After this I think we are ready to merge.